README.md

```diff
@@ -1,24 +1,298 @@
+- [x] Write clear, professional, and respectful review comments for other team members.
+- [x] Explain "why" a change is requested when giving a code review.
+- [x] Plan a 2+ week project with no interim Microverse-set milestones and submit it on time.
+- [x] Apply knowledge of setting working agreements to set group project teams up for success.
+- [x] Independently implement best practices for group agreements to improve teamwork in larger group project.
+- [x] Demonstrate ability to apply best practices of communication for resolving teamwork challenges.
+- [x] Understand that respect is the foundation of strong relationship-building with teammates.
+- [x] Show up throughout group projects as a reliable and committed team member who communicates and manages your time effectively.
+- [x] Recognize the importance of investing in good working relationships with teammates.
+- [x] Understand principles of strong teamwork (being reliable, committed, and consistent) and how to apply them in group projects.
+- [x] Recognize the value of making equal contributions to group projects to produce the best outcome.
+- [x] Use empty Kanban board to manage tasks with team and own time on the project.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>BackEnd</summary>
+  <ul>
+    <li><a href="https://railsguide.com/">Rails</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Test</summary>
+  <ul>
+    <li><a href="https://rspec.com/">RSPEC</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Delploy on render]** - Delploy on heroku
+- **[Improve security]** - Improve security
+- **[ Add testes ]** -  Add testes
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Version <a name="live-demo"></a>
+
+- [Comming soon]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Presentation -->
+
+## 🚀 Presenation video <a name="live-demo"></a>
+
+- [Coming soon]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+- [x] A web browser like Google Chrome.
+- [x] A code editor like Visual Studio Code with Git and Ruby.
+
+You can check if Git is installed by running the following command in the terminal.
+
+```
+$ git --version
+```
+
+Likewise for Ruby installation.
+
+```
+$ ruby --version && irb
+```
+
+### Install
+
+In the terminal, go to your file directory and run this command.
+
+```
+$ git clone https://github.com/melashu/hotel-room-reservation.git
+
+```
+
+$ gem install bundler
+
+```
+$ bundle install
+
+$ rails db:create db:migrate
+
```

Comment:
- [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍